### PR TITLE
fix No intellisense for Crash, Distribute, Push

### PR DESCRIPTION
**Pull request by Zongrenfeng**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
The previous missed add reference to the xml files inside nuspec files of crash, distribute, push. Also 3 project files need to be update too.

## Related PRs or issues
https://github.com/Microsoft/AppCenter-SDK-DotNet/pull/827/

### CODE REVIEW

File: nuget/MacAppCenterCrashes.nuspec [hunk 1]
```diff
@@ -22,16 +22,21 @@
 
     <!-- Portable -->
     <file src="$pcl_dir$/Microsoft.AppCenter.Crashes.dll" target="lib/portable-net45+win8+wpa81+wp8" />
+    <file src="$pcl_dir$/Microsoft.AppCenter.Crashes.xml" target="lib/portable-net45+win8+wpa81+wp8" />
 
     <!-- .NET Standard -->
     <file src="$netstandard_dir$/Microsoft.AppCenter.Crashes.dll" target="lib/netstandard1.0" />
+    <file src="$netstandard_dir$/Microsoft.AppCenter.Crashes.xml" target="lib/netstandard1.0" />
+    <file src="$netstandard_dir$/Microsoft.AppCenter.Crashes.dll" target="lib/netstandard1.0" />
```
Comment (MatkovIvan) [inline nuget/MacAppCenterCrashes.nuspec]: Duplicate

**Comment by guperrot** [comment]
@MatkovIvan do you know what that build error is?
> Attempting to build package from 'CAKE_SCRIPT_TEMPspec_copy.temp.nuspec'.
> Error NU5019: File not found: > 'CAKE_SCRIPT_TEMPAndroidAssemblies/Microsoft.AppCenter.Crashes.xml'.

I think the build definition had to be modified for other modules.

**Comment by MatkovIvan** [comment]
`DocumentationFile` tag is missed for `Release` configuration in `SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Microsoft.AppCenter.Crashes.Android.csproj`

File: scripts/utility.cake [hunk 2]
```diff
@@ -29,7 +29,10 @@ void CopyFiles(IEnumerable<string> files, string targetDirectory, bool clean = t
     }
     foreach (var file in files)
     {
-        CopyFile(file, targetDirectory + "/" + System.IO.Path.GetFileName(file));
+        if (FileExists(file))
```
Comment (MatkovIvan) [inline scripts/utility.cake]: Let's revert this to identify problems early

Comment (Zongrenfeng) [reply scripts/utility.cake]: Ok.

**Pull request merged**